src/Components/Header/Menu/Difficulty/DifficultyList.jsx

```diff
@@ -17,6 +17,6 @@ const DifficultyList = ({ difficultySetting, setDifficulty }) => {
 
 const mapStateToProps = ({ view: { difficulty } }) => ({ difficultySetting: difficulty });
 const mapDispatchToProps = dispatch => ({
-  setDifficulty: (difficulty, currentSlide) => dispatch(set_difficulty(difficulty, currentSlide))
-})
+  setDifficulty: (difficulty, currentSlide) => dispatch(setDifficulty(difficulty, currentSlide)),
```

Comment:
Use `bindActionCreators`. Much cleaner

ex:   dispatch => bindActionCreators({

Reply:
any chance you could give an example?